submissions/ArthurGorbenko/HTML5-Form/index.html

```diff
@@ -0,0 +1,487 @@
+<html lang="en">
+  <head>
+    <link
+      href="https://fonts.googleapis.com/css?family=Open+Sans&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="node_modules/normalize.css/normalize.css" />
+    <link rel="stylesheet" href="css/styles.css" />
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
```

Comment:
You have favicon in the assets but you not use it.

Reply:
Done